filament/src/details/Renderer.cpp

```diff
@@ -296,12 +296,9 @@ void FRenderer::skipFrame(uint64_t vsyncSteadyClockTimeNano) {
     // WARNING: while doing this we can't access any component manager
     auto& js = engine.getJobSystem();
 
-    auto *job = js.runAndRetain(jobs::createJob(js, nullptr, &FEngine::gc, &engine)); // gc all managers
+    js.runAndWait(jobs::createJob(js, nullptr, &FEngine::gc, &engine)); // gc all managers
 
     engine.flush();     // flush command stream
-
-    // make sure we're done with the gcs
-    js.waitAndRelease(job);
```

Comment:
does the ordering matter here?  Would you want to gc after the flush()?

Reply:
probably doesn't matter since before it was done in parallel.